package.json

```diff
@@ -46,7 +46,8 @@
     "bignumber.js": "^9.1.1",
     "cross-fetch": "4.0.0",
     "graphql": "^16.7.1",
-    "graphql-ws": "^5.14.0"
+    "graphql-ws": "^5.14.0",
+    "pako": "^2.1.0"
```

Comment:
Unused packages?

Reply:
it's needed for some of the functions inside the signingcosmwasmclient. Has an unzip function